chains/orchestrator-relays/runtime/dancelight/src/tests/common/xcm/mocknets.rs

```diff
@@ -104,17 +117,81 @@ decl_test_parachains! {
     }
 }
 
+// Store messages sent to ethereum throught the bridge
+thread_local! {
+    pub static ETH_BRIDGE_SENT_MSGS: RefCell<Vec<CommittedMessage>> = RefCell::new(Vec::new());
+}
+pub fn eth_bridge_sent_msgs() -> Vec<CommittedMessage> {
+    ETH_BRIDGE_SENT_MSGS.with(|q| (*q.borrow()).clone())
+}
+pub struct DancelightEthMockBridgeHandler;
+impl BridgeMessageHandler for DancelightEthMockBridgeHandler {
+    fn get_source_outbound_messages() -> Vec<BridgeMessage> {
+        // Get messages from the outbound queue
+        let msgs = DancelightRelay::ext_wrapper(|| {
+            snowbridge_pallet_outbound_queue::Messages::<<DancelightRelay as Chain>::Runtime>::get()
+        });
+
+        // Store messages in our static mock buffer
+        ETH_BRIDGE_SENT_MSGS.with(|sent_msgs| {
+            sent_msgs.borrow_mut().extend(msgs.clone());
+        });
+
+        // TODO: return the right Vec<BridgeMessage> here
+        Default::default()
```

Comment:
So this is not needed?

Reply:
I mean the TODO

Reply:
Then add it to the TODO: We don't check the dispatches messages from the bridge so it's fine to return Default::default here. Maybe explain why we can't do it now.